lib/workers/repository/update/branch/automerge.spec.ts

```diff
@@ -28,6 +29,13 @@ describe('workers/repository/update/branch/automerge', () => {
       expect(await tryBranchAutomerge(config)).toBe('no automerge');
     });
 
+    it('returns false if off schedule', async () => {
+      config.automerge = true;
+      config.automergeType = 'branch';
+      jest.spyOn(schedule, 'isScheduledNow').mockReturnValueOnce(false);
```

Comment:
the spy needs to be reset after each test, otherwise it will have side effects to other tests